service_mesh/v2x/ossm-threescale-webassembly-module.adoc

```diff
@@ -69,3 +69,10 @@ include::modules/ossm-threescale-webassembly-module-mapping-rule-object.adoc[lev
 include::modules/ossm-threescale-webassembly-module-examples-for-credentials-use-cases.adoc[leveloffset=+1]
 
 include::modules/ossm-threescale-webassembly-module-minimal-working-configuration.adoc[leveloffset=+1]
+
+[id="additional-resources_ossm-threescale-webassembly-module"]
```

Comment:
My apologies, 1 more thing, you don't actually need this ID for the additional resources section. 

This is also up to you, if you want, you can put this section so that it appears at the end of the module where it could be necessary. It looks like it would be right after line 39, if you wanted to switch it there. If you do move it, just change the `== Additional resources` to `.Additional resources. 

Thank you!

Reply:
Thanks a lot @kelbrown20 

I was in two minds to choose exactly your suggestion or do it the way I did. Appreciate the clarification now.